zulipterminal/cli/run.py

```diff
@@ -93,7 +93,7 @@ def fetch_zuliprc(zuliprc_path: str) -> None:
 
     while res.status_code != 200:
         print(in_color('red', "\nUsername or Password Incorrect!\n"))
-        res = get_api_key(realm_url)
+        res, email = get_api_key(realm_url)
```

Comment:
I've settled for `Tuple[Any, Str]`. I've seen your other comment about avoiding `Any`, but the actual type in this case is imported when the function is called, and I don't know if you'd rather change that or not.

As for the `run-mypy`, it does complain if there's incorrect assignment in line 91 but not in 95. I assume it's a matter control flow?

Reply:
I would include the type changes in this commit (much like ideally including tests for code when introducing the code), rather than in a separate later one.

Re `Tuple[Any, str]`, mypy considers each variable name to have one type, so that may be the issue you're seeing? Regarding adding the specific type, personally I would prefer to type this properly, and if that needs importing outside the function then that would be fine - do you agree, @amanagr ?

Reply:
> You can see why these two parts work well in the same commit?

Sure

Reply:
@neiljp I think it's fine.

Reply:
@vonknio Would you like to make that amendment now, given @amanagr 's response?